src/ProfileIOGroup.cpp

```diff
@@ -176,12 +177,12 @@ namespace geopm
 
         if (m_do_read[M_SIGNAL_REGION_HASH]) {
             for (int idx = 0; idx < m_num_cpu; ++idx) {
-                m_per_cpu_sample[M_SIGNAL_REGION_HASH][idx] = hash_to_signal(m_application_sampler.cpu_region_hash(idx));
```

Comment:
This change will impact how hash invalid and hint inactive are reported during normal use.  Why do we want this change?